scrapy/http/request/json_request.py

```diff
@@ -0,0 +1,28 @@
+"""
+This module implements the JSONRequest class which is a more convenient class
+(than Request) to generate JSON Requests.
+
+See documentation in docs/topics/request-response.rst
+"""
+
+import json
+
+from scrapy.http.request import Request
+
+
+class JSONRequest(Request):
+    def __init__(self, *args, **kwargs):
+        if 'method' not in kwargs:
+            kwargs['method'] = 'POST'
```

Comment:
Maybe this is a slightly controversial thing to default to POST method, I think it makes sense and ``XmlRpcRequest`` does this too, but would like someone else's opinion.

Reply:
I inspected some (~20) existing spiders/websites and, quite logically, POST was always used when any data was sent in the request and otherwise (when using only query string, no data sent) it's GET. So maybe it should depend on `data`?